librosa/core/pitch.py

```diff
@@ -417,8 +417,8 @@ def _parabolic_interpolation(y_frames):
     return parabolic_shifts
 
 
-def yin(y, sr=22050, frame_length=2048, win_length=1024, hop_length=None,
-        fmin=None, fmax=880, peak_threshold=0.1, center=True, pad_mode='reflect'):
+def yin(y, fmin, fmax, sr=22050, frame_length=2048, win_length=1024, hop_length=None,
+        peak_threshold=0.1, center=True, pad_mode='reflect'):
     '''Fundamental frequency (F0) estimation. [1]_
 
     .. [1] De Cheveigné, A., & Kawahara, H. (2002).
```

Comment:
Some updates to the formatting for references:

1. Use `[#]_`  and `.. [#] ....` instead of explicit numbers.  This triggers auto-numbering of footnotes so that we don't end up with hundreds of "reference #1" in the index
2. Don't use reference markers `[#]_` in the first line of the docstring.  This messes up the top-level index in the documentation, see this [test build](https://librosa.org/doc/main/core.html#spectral-representations) for an example of what I mean.
3. Reference entry should be complete; I've been using MLA style generated by google scholar most of the time.